runtime/plugin/man.lua

```diff
@@ -8,7 +8,7 @@ vim.api.nvim_create_user_command('Man', function(params)
   if params.bang then
     man.init_pager()
   else
-    local err = man.open_page(params.count, params.smods, params.fargs)
+    local _, err = pcall(man.open_page, params.count, params.smods, params.fargs)
```

Comment:
open_page may return a string error: https://github.com/neovim/neovim/blob/ad337025314a424f19f96decb69b6a9d36121efe/runtime/lua/man.lua#L685

but you have now assigned that to `_`.

Reply:
d'oh